### DISCUSSION TITLE
vc-overlay-dynamic这个组件在线运行后点击跟随按钮后报错

### DISCUSSION THREAD

**Discussion by ytl1530** [zouyaoji/vue-cesium#456] [General]
应该是这个方法
 dynamicOverlayRef.value.trackOverlay(0, {
          mode,
          viewFrom: [0, 0, 1800]
})
![image](https://github.com/zouyaoji/vue-cesium/assets/90435163/2730169c-5f9d-4572-be25-46dc16d037db)

**Comment by zouyaoji** [comment]
你好，我试了一下，没能重现哈。

**Reply by ytl1530** [reply]
那请问一下你用的cesium是哪个版本

**Reply by ytl1530** [reply]
找到问题了，数据原因
